PowerEditor/src/ScintillaComponent/AutoCompletion.h

```diff
@@ -88,7 +88,65 @@ class AutoCompletion {
 	generic_string _keyWords;
 	size_t _keyWordMaxLen = 0;
 
-	FunctionCallTip _funcCalltip;
+    const char* xpmfn[53] = {

```

Comment:
FYI: You can declare it as `const char* xpmfn[] =`. This way it will compute the array length from the initializer list.

Reply:
I originally had that, but got a compiler error:

```
E0070 incomplete type is not allowed
```

Reply:
Okay, I see. 
Maybe it's better to move this declaration to `AutoCompletion.cpp` since it's used only there.
This way the header file won't be cluttered with a huge XPM string.

Reply:
Done!  Thanks again.